rclpy/rclpy/qos_event.py

```diff
@@ -242,20 +242,6 @@ def create_event_handlers(
                 callback=self.incompatible_qos,
                 event_type=QoSSubscriptionEventType.RCL_SUBSCRIPTION_REQUESTED_INCOMPATIBLE_QOS,
                 parent_handle=subscription_handle))
-
-        if self.liveliness:
-            event_handlers.append(QoSEventHandler(
-                callback_group=callback_group,
-                callback=self.liveliness,
-                event_type=QoSSubscriptionEventType.RCL_SUBSCRIPTION_LIVELINESS_CHANGED,
-                parent_handle=subscription_handle))
-
-        if self.message_lost:
-            event_handlers.append(QoSEventHandler(
-                callback_group=callback_group,
-                callback=self.message_lost,
-                event_type=QoSSubscriptionEventType.RCL_SUBSCRIPTION_MESSAGE_LOST,
-                parent_handle=subscription_handle))
         elif self.use_default_callbacks:
```

Comment:
This diff is hard to read.
The `elif` block is applying to a different `if` now.